examples/notebooks/set_x0_sn_catalog.ipynb

```diff
@@ -0,0 +1,1338 @@
+       "2095   -19.477398\n",
+       "2096   -19.477398\n",
+       "2097   -19.113095\n",
+       "2098   -19.113095\n",
+       "2099   -18.894093\n",
+       "2100   -18.894093\n",
+       "2101   -18.894093\n",
+       "2102   -18.894093\n",
+       "2103   -19.645625\n",
+       "2104   -19.645625\n",
+       "2105   -19.228412\n",
+       "2106   -19.228412\n",
+       "2107   -19.183008\n",
+       "2108   -19.183008\n",
+       "2109   -19.064679\n",
+       "2110   -19.064679\n",
+       "2111   -19.218430\n",
+       "2112   -19.218430\n",
+       "2113   -19.481811\n",
+       "2114   -19.481811\n",
+       "2115   -19.703833\n",
+       "2116   -19.703833\n",
+       "2117   -19.115300\n",
+       "2118   -19.115300\n",
+       "2119   -19.406362\n",
+       "2120   -19.406362\n",
+       "2121   -19.406362\n",
+       "2122   -19.369897\n",
+       "2123   -19.369897\n",
+       "Name: MB, Length: 2124, dtype: float64"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dc2_df['MB']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "#DC2 Cosmology\n",
+    "#FlatLambdaCDM(H0=71 km / (Mpc s), Om0=0.265, Tcmb0=0 K, Neff=3.04, m_nu=None, Ob0=0.0448)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "source = sncosmo.get_source('salt2-extended')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "model = sncosmo.Model(source=source)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['z', 't0', 'x0', 'x1', 'c']"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.param_names"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([ 0.,  0.,  1.,  0.,  0.])"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Danny's x1\n",
+    "model.set(x1 = 1.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 49,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "cosmo = FlatLambdaCDM(H0=71, Om0=0.265, Tcmb0=0, Neff=3.04, m_nu=None, Ob0=0.0448)"
```

Comment:
Thanks Bryce ! Can you please add this as a variable like `DC2Cosmo` so that this is not hard coded. 
Also, it is likely that the cut paste I did on slack did not work accurately.